cinema/admin.py

```diff
@@ -1,13 +1,7 @@
 from django.contrib import admin
 
 from .models import (
-    CinemaHall,
-    Genre,
-    Actor,
-    Movie,
-    MovieSession,
-    Order,
-    Ticket,
+    Actor, CinemaHall, Genre, Movie, MovieSession, Order, Ticket
```

Comment:
If importing more than two items, it is better to write them on separate lines. This improves readability and maintenance of the code. 

Example:

From taxi.models import ( 
	Driver,
	Manufacturer,
	Car
)
Read more about topic [here](https://peps.python.org/pep-0008/#imports)